README.md

```diff
@@ -1,19 +1,27 @@
 # shifthelper_webinterface
 The webinterface for the shifthelper
 
+## Technology stack
 
-# how to start:
+We are using
 
- - clone
- - cd into 
+* twitter bootstrap for responsivness (make it look good on mobile)
+* react (without jsx) for the ui component
```

Comment:
what is jsx? This? https://jsx.github.io/

what do you mean by: "the ui component?"

Reply:
See https://reactjs.org/docs/introducing-jsx.html

UI is basically only the alert table at the moment